app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
```

Comment:
[OPTIONAL]

- What do you think about adding the `dependent: :destroy` option to the `has_many` associations? This will help to make sure that all associated objects are deleted when a `Post` gets deleted.

You can learn more about this [here](https://guides.rubyonrails.org/association_basics.html#dependent)